pkgs/tools/games/gamemode/default.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, libgamemode32
+, meson
+, ninja
+, pkg-config
+, dbus
+, inih
+, systemd
+, appstream
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gamemode";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "FeralInteractive";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
```

Comment:
Repo has no submodules.

Reply:
Right, thanks :slightly_smiling_face: I'm not sure why I added that...